locale/en/locale.cfg

```
@@ -63,38 +63,44 @@ map_info_sub_caption=    ..diggy diggy choo choo..
-message_danger3=Robot voice: INTRUDER ALERT! Lifeforms detected! Must eliminate!
-message_danger4=Robot voice: Nuclear missiles armed, launch countdown enabled.
-message_jump180=Comfylatron: Acumulator charging disabled, 180 seconds countdown to jump!
-message_jump60=Comfylatron: ChronoTrain nearly charged! Grab what you can, we leaving in 60 seconds!
-message_jump30=Comfylatron: You better hurry up! 30 seconds remaining!
+map_info_text_old=Comfylatron has seized the Fish Train and turned it into a time machine.\n Your job as his slave is:\n\n[1] Keep train alive at all costs\n[2] Gather resources while travelling through different maps.\n[3a] Press enter on cargo wagons to enter insides of train.\n[3b] Press enter on cars to exit the train.\n[4] Charging acumulators inside train speeds up jumps when leaving early is needed.\nCharging and jumping creates huge pollution so be ready to get attacked.\n[5] Items inserted into comfylatron chests get teleported into train.\n[6] Some planets are poor, some are rich, and some are just too dangerous.\n\n Loot, but also evolution grows with jumps performed.\n During jump, personnel will be teleported in,\n however dead bodies nor buildings won't.\nMining productivity grants inventory space and handmining speed.\n\nGood luck. Don't let biters ruin the show!
+
+map_info_text=Comfylatron has seized the Fish Train and turned it into a time machine.\n Your job as his slave is:\n\n[1] Keep train alive at all costs.\n[2] Gather resources while travelling through different planets.\n[3a] Press enter on cargo wagons to enter the insides of train.\n[3b] Press enter on cars to exit the Chronotrain.\n[4] Charging accumulators inside the Chronotrain speeds up the jump, but creates huge pollution that annoys the biters.\n[5] Items inserted into the Chronotrain chests get teleported into the train.\n[6] Some planets are poor, some are rich, and some are just too dangerous.\n\nLoot, but also evolution, grows with jumps performed.\n During jump, personnel and their inventories will be teleported in, but anything left behind outside won't.\nEarly jumps award 20 coins for each minute spared (until the 20th jump).\nObtaining mining productivity research grants inventory space and hand-mining speed.\n\nGood luck. Don't let biters ruin the show!
+planet_jump=Destination: __1__, Ore richness: __2__, Day length: __3__
+
+message_danger1=Comfylatron: The chronojump misfired! Hang on a second!
+message_danger2=Comfylatron: Oo-ee! That's worse than I thought, our chronojump went sideways..
+message_danger3=Robot voice: INTRUDER ALERT! Lifeforms detected!! Must eliminate!!
+message_danger4=Comfylatron: Welp. The charger is shot. We'll have to wait.
+message_danger5=Robot voice: Nuclear missiles armed. Launch countdown enabled.
+message_rampup50=Comfylatron: The biters are making me nervous... =owo=
```

```suggestion
message_rampup50=Comfylatron: The biters seem to start getting stronger pretty fast!
```
tells players absolutely nothing about what is happening.

---

That is intentional